mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/options/writeconcern/WriteConcernPropagationTest.java

```diff
@@ -114,7 +114,7 @@ public void shouldApplyConfiguredWriteConcernForRemoveTuple() {
 		session.close();
 
 		// then expect a call to remove with the configured write concern
-		verify( mockClient.getCollection( "GolfPlayer" ).withWriteConcern( WriteConcern.MAJORITY ) ).deleteMany( any( Document.class ) );
+		verify( mockClient.getCollection( "GolfPlayer" ).withWriteConcern( WriteConcern.MAJORITY ) ).findOneAndDelete( any( Document.class ) );
```

Comment:
How this get affected by the PR?

Reply:
I need check deleted document. Do the document have content in GridFS
Document deleted = collection.findOneAndDelete( toDelete );
		if ( deleted != null ) {
			GridFsUtil.removeFromGridFsByEntity( getCurrentDB(), optionService, deleted, key );
}